GCTC-NTGC/gc-digital-talent
# Accessibility - accessible edit links

Resolves #3830 

## Notes
- Fixes the ["Links must have discernible text"](https://dequeuniversity.com/rules/axe/4.4/identical-links-same-purpose?application=AxeChrome) accessibility issue on all tables with "Edit" button.

## Testing 
- Go to [admin app](http://localhost:8000/en/admin/dashboard) and login
- Open each table and run the axe dev tools scanner. Ensure that the error above does not show up!
